source/MaterialXView/Material.cpp

```diff
@@ -516,7 +516,10 @@ void Material::bindLights(mx::LightHandlerPtr lightHandler, mx::GLTextureHandler
                 // Bind any associated uniforms.
                 if (pair.first == "u_envRadiance")
                 {
-                    _glShader->setUniform("u_envRadianceMips", desc.mipCount);
+                    if (_glShader->uniform("u_envRadianceMips", false) != -1)
```

Comment:
Viewer would assert if we did not use FIS for env mapping. I need to add in a user option for FIS vs pre-filtering. I'll check in with you tomorrow as the option you did for OSL is only pre-filtering (let's get that in first :)).